nautobot/extras/models/models.py

```diff
@@ -822,13 +824,51 @@ def enqueue_job(cls, func, name, obj_type, user, *args, schedule=None, **kwargs)
         user: User object to link to the JobResult instance
         args: additional args passed to the callable
         schedule: Optional ScheduledJob instance to link to the JobResult
-        kwargs: additional kargs passed to the callable
+        celery_kwargs: Dictionary of options to pass as kwargs to Celery `apply_async` method
```

Comment:
```suggestion
        celery_kwargs: Dictionary of options to pass as **kwargs to Celery `apply_async` method
```

Since celery accepts an argument named `kwargs` that are not **kwargs, I think it is necessary to specify `**kwargs` to be clear on how this is actually being passed